packages/amplication-data-service-generator/src/server/package-json/package.template.json

```diff
@@ -12,7 +12,7 @@
     "db:migrate-save": "prisma migrate dev",
     "db:migrate-up": "prisma migrate deploy",
     "db:clean": "ts-node scripts/clean.ts",
-    "db:init": "run-s \"db:migrate-save -- --name 'initial version'\" db:migrate-up seed",
```

Comment:
I think we should keep the seed. instead we need to remove the logic about the user from the seed (although it is the only data in the seed right now)